dot/state/block_notify.go

```diff
@@ -75,7 +75,7 @@ func (bs *BlockState) notifyImported(block *types.Block) {
 		return
 	}
 
-	logger.Trace("notifying imported block chans...", "chans", bs.imported)
+	logger.Trace("notifying imported block channels...")
```

Comment:
I removed logging channels here since it would just log out a pointer, I'm not sure this is useful.